engine/src/main/java/org/terasology/rendering/opengl/ScreenGrabber.java

```diff
@@ -88,9 +94,8 @@ public void takeScreenshot() {
      * to be written to file, by submitting a task to the ThreadManager. The task is then executed as soon as possible
      * but not necessarily immediately.
      *
-     * The file is then saved in the designated screenshot folder with a filename in the form:
-     *
-     *     Terasology-[yyMMddHHmmss]-[width]x[height].[format]
+     * The file is then saved in the designated screenshot folder,
+     * or if it's a game preview image, it'll be placed in the save game folder.
```

Comment:
Maybe use a Javadoc style `@link` here to direct the reader to the related class for that logic.  May also be able to shorten the text a bit to still fit on one line :-)